batch-index/src/test/java/no/unit/nva/search/StubS3Driver.java

```diff
@@ -0,0 +1,42 @@
+package no.unit.nva.search;
+
+import static nva.commons.core.attempt.Try.attempt;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.zip.GZIPInputStream;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.ioutils.IoUtils;
+import software.amazon.awssdk.services.s3.model.NoSuchKeyException;
+
+public class StubS3Driver extends S3Driver {
```

Comment:
I do not know why this re-appeared as a new file. I think it is the different packaging in combination with the renaming